installer/conf/omsagent.d/npmd.conf

```diff
@@ -0,0 +1,14 @@
+<source>
+    type npmd
+</source>
+
+<match npmd.oms.blob>
```

Comment:
Have you tested that this works? where is this tag defined? is this an existing tag?